plutus-core/plutus-ir/test/TransformSpec.hs

```diff
@@ -25,6 +27,7 @@ transform = testNested "transform" [
     , nonStrict
     , letFloat
     , inline
+    , beta
```

Comment:
and deadcode!!

Reply:
Ah!  I swear I added it...  Might have accidentally stashed it somewhere.

Reply:
odd... test output contains the results of `deadCode` ..., but the file didn't...